tests/test_compiler.py

```diff
@@ -45,6 +46,9 @@ def test_compile_success():
       compiled_abspath = os.path.join(root, compiled)
       with open(compiled_abspath, 'rb') as fp:
         fp.read(4)  # Skip the magic header.
+        if sys.version_info[:2] >= (3, 7):
```

Comment:
consider adding `compatibility.PY37`, so this would be the more readable `if compatibility.PY37:`

Reply:
I was reluctant to do that since compatibility is a prod module and the only use would be here in a test.

Reply:
That sounds fine to keep as is.

If we were to make the change, we would also need to add a variable for every major version, e.g. `PY34` and `PY35`, which is outside the scope of this PR.